hotdoc/utils/utils.py

```diff
@@ -160,10 +167,30 @@ def __get_extra_extension_classes(paths):
 
     return extra_classes
 
+def __load_extra_extension_moudles(paths: T.List[str]) -> T.List[T.Type['Extension']]:
+    """
+    Banana banana
+    """
+    extra_classes = []
+    for p in [Path(x) for x in paths]:
+        if not p.exists() or not p.is_file():
+            info(f'Extension {p} does not exist or is not a file (skipping)')
```

Comment:
use `warn` instead, will also cause `--fatal-warnings` to error out in that case, which seems appropriate

Reply:
I also changed the `info` in `__get_extra_extension_classes` into a `warn`.